build.gradle

```diff
@@ -0,0 +1,112 @@
+
+// This gradle build depends on Mendix internal resources, the lastest commit
+// which did not is 6e1527a5e31347c6914bc34deab31ef4ab46953b. You may restore
+// the gradle scripts to that commit if you which to use them at your own risk.
+// We don't guarantee that those will keep working in the future.
+
+import com.github.jk1.license.render.*
+
+plugins {
+    id 'java'
+    id 'com.mendix.gradle.publish-module-plugin' version '1.10'
+    id 'net.researchgate.release' version '2.8.1'
+    id 'com.github.jk1.dependency-license-report' version '2.0'
+}
+
+sourceCompatibility = '11'
+[compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
+
+project.ext {
+    mendixPublicApiVersion = '8.12.5'
+}
+
+mxMarketplace {
+    appNumber = 37114
+    appName = 'ObjectHandling'
+    moduleName = 'CommunityCommons_ObjectHandling'
```

Comment:
Should be just `'ObjectHandling'` (the module name in the .mpr).

Reply:
Fixed.